test/powershell/Modules/Microsoft.PowerShell.Utility/Environment-Variables.Tests.ps1

```diff
@@ -40,4 +40,10 @@ Describe "Environment-Variables" -Tags "CI" {
 	$ENV:TESTENVIRONMENTVARIABLE | Should Be $expected
 
     }
+
+    It "User Agent should reflect current plafformID" {
+    $bingdings = [System.Reflection.BindingFlags]::NonPublic -bxor [System.Reflection.BindingFlags]::Static
+    $platform = [Microsoft.PowerShell.Commands.PSUserAgent].GetProperty('Platform',$bingdings).GetValue($null,$null)
```

Comment:
We should probably not use reflection. Instead we should make a web call and ensure the `User-Agent` request header contains the desired data. 

https://github.com/PowerShell/PowerShell/blob/b07f24e3c9fec8bdeee0f70cb31edbbff0e29b9b/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1#L456-L467